packages/cli/src/util/read-json.ts

```diff
@@ -0,0 +1,15 @@
+import { CantParseJSON } from './errors-ts';
+
+export default function readJSON<T>(
+  jsonRaw: string
+): T | undefined | CantParseJSON {
+  if (!jsonRaw) {
+    return;
+  }
+
+  try {
+    return JSON.parse(jsonRaw);
+  } catch (error: any) {
+    return new CantParseJSON(error.message, jsonRaw);
+  }
+}
```

Comment:
This new function was added because the existing `readJSONFile` reads an actual file.

We could make that function use this one, but the errors thrown are different. It didn't seem worth the minimal re-use.